packages/@react-types/menu/src/index.d.ts

```diff
@@ -51,7 +51,7 @@ export interface SpectrumMenuTriggerProps extends MenuTriggerProps {
   children: ReactElement[]
 }
 
-export interface MenuProps<T> extends CollectionBase<T>, MultipleSelection {
+export interface MenuProps<T> extends CollectionBase<T>, MultipleSelection, FocusEvents {
```

Comment:
i think changes in this file were unintentional?

Reply:
I added that when I first started with trying to account for focus within the Breadcrumbs, but once realizing that I only need to handle the case where a menu item is selected, this can be removed.